vtexdocs/RapiDoc
# Fix/request on page load

#### What is the purpose of this pull request?

To remove the request made after the page loads.

#### What problem is this solving?

The first request always returns an error.

#### How should this be manually tested?

Open any API Reference page on the [preview](https://deploy-preview-199--elated-hoover-5c29bf.netlify.app/) and check if the request response is empty.

#### Screenshots or example usage

#### Types of changes
